### PR TITLE
Small improvements

### PR DESCRIPTION
**Pull request by oterral**
This PR does 2 things:

 - Add an anchor the marker used in draw.
 - Deactivate correctly the query tool in 3d (only src mode)
 - Constraint the center of the map

[Test](//mf-geoadmin3.int.bgdi.ch/teo_imp/index.html)

### CODE REVIEW

**Comment by oterral** [comment]
jenkins test

File: src/components/search/SearchDirective.js [hunk 1]
```diff
@@ -170,14 +174,14 @@ goog.require('ga_what3words_service');
                 if (res && res.geometry && res.geometry.lng &&
                     res.geometry.lat) {
                   var newPos = ol.proj.transform([res.geometry.lng,
-                    res.geometry.lat],
-                  'EPSG:4326', 'EPSG:21781');
+                    res.geometry.lat], 'EPSG:4326',
+                    $scope.map.getView().getProjection());
```
Comment (loicgasser) [inline src/components/search/SearchDirective.js]: good preparation

**Comment by loicgasser** [comment]
rebase your branch on master, test link is not very convincing :D

**Comment by loicgasser** [comment]
code LGTM

**Comment by oterral** [comment]
fixed

File: src/js/MainController.js [hunk 2]
```diff
@@ -68,6 +68,7 @@ goog.require('ga_window_service');
         renderer: 'canvas',
         view: new ol.View({
           projection: defaultProjection,
+          extent: gaMapUtils.defaultExtent,
```
Comment (loicgasser) [inline src/js/MainController.js]: crazy

**Comment by oterral** [comment]
thx for the review

**Pull request merged**